### PR TITLE
Fix: Fix text transformation being reset when changing text parameters

### PR DESCRIPTION
**Pull request by meiliang1123**

### CODE REVIEW

**Comment by meiliang1123** [comment]
JIRA: SB-17

File: src/app/models/SVGActionsFactory.js [hunk 1]
```diff
@@ -1356,14 +1356,20 @@ class SVGActionsFactory {
...
             sourceWidth: width,
             sourceHeight: height,
             width,
             height,
             transformation: {
-                width: width * model.transformation.scaleX,
-                height: height * model.transformation.scaleY
+                positionX: logicalX,
+                positionY: logicalY,
+                width: width * svgModel.scaleX,
```
Comment (parachvte) [inline src/app/models/SVGActionsFactory.js:1368]: negative?

**Review by bbaa77770 — APPROVED** [review]
LGTM

**Review by parachvte — APPROVED** [review]

**Comment by parachvte** [comment]
picked to branch `bugfix-v3.14`.

**Pull request merged**